src/lfortran/parser/semantics.h

```diff
@@ -1559,6 +1559,30 @@ void add_ws_warning(const Location &loc,
                         {loc},
                         "help: write this as 'integer(8)'");
                 }
+        } else if (end_token == yytokentype::KW_COMPLEX) {
+                if(a_kind == 4){
+                        diagnostics.parser_style_label(
+                        "Use complex(4) instead of complex*4",
+                        {loc},
+                        "help: write this as 'complex(4)'");
+                } else{
+                        diagnostics.parser_style_label(
+                        "Use complex(8) instead of complex*8",
+                        {loc},
+                        "help: write this as 'complex(8)'");
+                }
```

Comment:
```suggestion
                        std::string kind = std::to_string(a_kind);
                        diagnostics.parser_style_label(
                        "Use complex(" + kind + ") instead of complex*" + kind,
                        {loc},
                        "help: write this as 'complex(" + kind + ")'");
```
This would work for all the kind cases.
Please update the same for other types.

Reply:
Yes it makes sense. Thanks.